pkg/mutation/mutators/copy.go

```diff
@@ -0,0 +1,55 @@
+package mutators
+
+import "fmt"
+
+// deepCopy copies the output of json.Unmarshal when used on a
+// map[string]interface{}. It is not meant for generic copying of arbitrary Go
+// values.
+//
+// If deepCopy returns an error, it either indicates the caller passed an
+// invalid argument or deepCopy() has a bug/is not exhaustive.
+func deepCopy(in interface{}) (interface{}, error) {
```

Comment:
Could we use `runtime.DeepCopyJSONValue()`?

Reply:
Nice!